apps/vr-tests-react-components/src/stories/ProgressBar.stories.tsx

```diff
@@ -1,8 +1,11 @@
 import * as React from 'react';
 import { storiesOf } from '@storybook/react';
 import { ProgressBar } from '@fluentui/react-progress';
+import { PauseAnimationDecorator, TestWrapperDecoratorFullWidth } from '../utilities/TestWrapperDecorator';
 
 storiesOf('ProgressBar converged', module)
+  .addDecorator(PauseAnimationDecorator)
+  .addDecorator(TestWrapperDecoratorFullWidth)
```

Comment:
Based on the vr-tests, you should add a fixed width or max width decorator instead of this one and crop to the `testWrapper`

```suggestion
  .addDecorator(story => (
    <div className="testWrapper" style={{ maxWidth: '750px' }}>
      {story()}
    </div>
  ))
  .addDecorator(story => (
    <StoryWright steps={new Steps().snapshot('default', { cropTo: '.testWrapper' }).end()}>
      {story()}
    </StoryWright>
  ))
```

Reply:
PauseAnimationDecorator is meant to be used together with TestWrapperDecoratorFixedWidth (which adds the test wrapper class and sets the width). See [my comment](https://github.com/microsoft/fluentui/pull/27156/files#r1183135860) below, for a suggestion.